### DISCUSSION TITLE
Duplicazione entità con variazione di segno

### DISCUSSION THREAD

**Discussion by bogontom** [indomus/forum#3022] [HUB: Home Assistant]
Salve a tutti, da poco sono utilizzatore di Home Assistant e pian piano sto facendo esperienza nella programmazione yaml.
La mia abitazione è già munita impianto domotico knx con webserver domotica labs ma volevo integrarlo con home assistant considerato gli indubbi vantaggi.
All' interno del file yaml ho configurato l' acquisizione di alcuni registri modbus , uno dei quali è un registro con segno che varia in base alla potenza in ingresso o uscita dall' abitazione. Tale valore è negativo quando sto acquistando energia da Enel , mentre diventa positivo quanto sto vendendo energia ad Enel.
Detto ciò sarebbe mia intenzione, partendo da questo registro , creare 2 entità : la prima " potenza prelevata " quando questo valore è inferiore a 0 e la seconda " potenza immessa " quando questo valore risulta positivo. Potete aiutarmi gentilmente. Grazie

Ps. Spero di aver postato il tutto correttamente. in caso contrario correggetemi senza ritegno..

**Comment by Scialla** [comment]
consiglio di usare `filter` per creare le due entità, usando rispettivamente i due filtri `lower_bound` per escludere i valori negativi sotto lo `0`) e `upper_bound` per escludere i valori positivi
https://www.home-assistant.io/integrations/filter/
